apps/web/src/app/nft/[chain]/[address]/[token]/page.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
+
```

Comment:
we don't need this extra space after `'use client';`, fix on all possible places